src/main/java/igrad/ui/StatusBar.java

```diff
@@ -0,0 +1,40 @@
+package igrad.ui;
```

Comment:
good job, I needed this status bar too, to display the course info, like name, etc. thanks